app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumFragment.java

```diff
@@ -1,7 +1,10 @@
 package com.fstyle.androidtrainning.screen.main.mainfragments.online.subfragment.topalbum;
 
+import android.content.Context;
```

Comment:
ERROR: Unused import: android.content.Context. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports